### ISSUE TITLE
When installing noobaa, using operator version that has a different version of CRD the error is not clear

### ISSUE THREAD

**Issue by liranmauda** [noobaa/noobaa-operator#56]
When installing noobaa, using operator version that has a different version of CRD the error is not clear.

If we installed noobaa on openshift and we used operator version 1.1.1, then we changed the operator cli version to 1.2.0, we will stay on "INFO[0152] ⏳ System Phase is "Configuring". Waiting for phase ready ..."

**Comment by guymguym** [comment]
Right, however the CRDs are still alpha which means we make changes to it without backward compatibility.

**Comment by nimrod-becker** [comment]
I think it's more about giving the info for troubleshooting than actual
backward compatability

**Comment by guymguym** [comment]
CRD's are the same for all namespaces on the cluster so either need separate clusters when not in sync with other users of the operator, or need to support a "namespaced" version of the CRD's - for example by replicating them with the namespace prefixed to their name (need to adjust the plural too for that)
